ipengine_config.py

```diff
@@ -407,10 +393,10 @@
 # Configurable for MPI initialization
 
```

Comment:
I forgot we had this! I'm sure this case is exactly why.

I think startup files should be the easier way to go, but it's possible that engines aren't running the startup files. Are you still running IPython 2.3 on spirit, or 3.x?